x-pack/plugin/core/template-resources/src/main/resources/entsearch/connector/chat/comments-template.json

```diff
@@ -0,0 +1,46 @@
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "comments": {
+          "type": "nested",
+          "properties": {
+            "id": {
+              "type": "keyword"
+            },
+            "author": {
```

Comment:
I have a feeling that author/owner will be common enough to have a global definition? Could we have e.g. `user-definition-template`? and this can be reused in:
- base doc template for owner
- comments template for author
- support case for customer (extended with org id etc)